doc/source/getting-started.rst

```diff
@@ -0,0 +1,79 @@
+:tocdepth: -1
```

Comment:
this is a template example file we want to remove it.  Also the skpkg image below.   This is one reason I want the `package update public`